src/org/zaproxy/zap/extension/proxies/ProxiesAPI.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2017 The ZAP Development team

```

Comment:
team should have a capital, noticed this in other classes in this PR too

Reply:
Ah, copy and paste ;) Will aim to fix them all.